visibility.md

```diff
@@ -1,7 +1,70 @@
 # Code of Conduct visibility
 
-- Advise from our transparency report on how to make CoC visible
-- Guidance on what should be on the CoC page
-- Guidance on what should be on the posters
-- Guidance where to link to CoC page
+One of the most important things to do before and during the event is 
+making sure that everyone knows about existence of Code of Conduct and 
+have no problems to find details about it. Failing to communicate about it 
+might cause unnecessary discussions in case of problems or might discourage 
```

Comment:
I'd replace "might cause unnecessary discussions in case of problems" with "might cause people to not report occuring incidents", which I think is the biggest problem with lack of visibility.